doc/external-signer.md

```diff
@@ -63,29 +65,59 @@ Under the hood this uses a [Partially Signed Bitcoin Transaction](psbt.md).
 $ bitcoin-cli -rpcwallet=<wallet> sendtoaddress <address> <amount>
 ```
 
-This prompts your hardware wallet to sign, and fail if it's not connected. If successful
-it automatically broadcasts the transaction.
+This constructs a PSBT and prompts your external signer to sign (will fail if it's not connected). If successful, Bitcoin Core finalizes and broadcasts the transaction.
 
 ```sh
-{"complete": true, "txid": <txid>}
+{"complete": true, "txid": "<txid>"}
 ```
 
 ## Signer API
 
-In order to be compatible with Bitcoin Core any signer command should conform to the specification below. This specification is subject to change. Ideally a BIP should propose a standard so that other wallets can also make use of it.
+More recent versions of Bitcoin, around v29 and later, have made significant improvements. It is recommended to use more recent versions.
+
+In order to be compatible with Bitcoin Core, any signer command should conform to the specification below. This specification is subject to change. Ideally a BIP should propose a standard so that other wallets can also make use of it.
 
 Prerequisite knowledge:
 * [Output Descriptors](descriptors.md)
 * Partially Signed Bitcoin Transaction ([PSBT](psbt.md))
 
+### Flag `--chain <name>` (required)
+
+With `<name>` one of `main`, `test`, `signet`, `regtest`.
+
+Previously, flag `--testnet` would be used to indicate "_testnet_" specifically. Presently, all calls of external-signer include `--chain` argument with value.
```

Comment:
This explanation of previous behaviour seems more appropriate for release notes.